js/sykmeldinger/data/sm-sykmeldinger/smSykmeldinger.js

```diff
@@ -2,7 +2,64 @@ import { createReducer } from '../../../reducers/createReducer';
 import {
     HENTER_SM_SYKMELDINGER,
     HENT_SM_SYKMELDINGER_FEILET,
-    SM_SYKMELDINGER_HENTET } from './smSykmeldingerActions';
+    SM_SYKMELDINGER_HENTET,
+    BEKREFTER_LEST_SM_SYKMELDING,
+    SM_SYKMELDING_BEKREFTET_LEST,
+    SM_SYKMELDING_BEKREFT_LEST_FEILET, KVITTERING_VIST_LENGE_NOK,
+} from './smSykmeldingerActions';
+
+const spesialHandler = (state, action) => {
```

Comment:
Kan denne få et litt mer beskrivende navn? Det er vel en eller annen form for `avvist_sykmelding_bekreftet` eller noe sånt?

Reply:
Dette er egentlig en funksjon man kan sende inn som en utvidelse av `createReducer`-funksjonen, slik at den kan håndtere mer spesifikke caser enn HENTER_XX/HENTET_XX/HENTING_XX_FEILET. Har du forslag til navn, @knutnyg ?

Reply:
Skjønner! Da er ikke navngivningen så gæren allikevel synes jeg 👍